src/Confluent.Kafka/Producer.cs

```diff
@@ -341,12 +344,13 @@ public int Poll(int millisecondsTimeout)
         ///     Returns the number of events served.
         /// </returns>
         public int Poll(TimeSpan timeout)
-            => Poll(timeout.TotalMillisecondsAsInt());
-
+            => Poll(timeout.TotalMillisecondsAsInt());
+
         /// <summary>
-        ///     Poll for callback events. Only use this method on producer
-        ///     instances where background polling is disabled. Blocks
-        ///     until there is a callback event ready to be served.
+        ///     Poll for callback events. You will not typically need
```

Comment:
i've just discovered you can reference text blocks in an external file - we should probably use this approach where it makes sense.

it's very desirable to have the docs on every method from an intellisense point of view at least. but yes the duplication is highly undesirable, I don't like it.